debug.go

```diff
@@ -115,6 +115,9 @@ func describeRequest(op interface{}) (s string) {
 		addComponent("offset %d", typed.Offset)
 		addComponent("length %d", typed.Length)
 		addComponent("mode %d", typed.Mode)
+
+	case *fuseops.ReleaseFileHandleOp:
+		addComponent("Handle %d", typed.Handle)
```

Comment:
lowercase “handle” for consistency please

Reply:
Done.